app/models/manageiq/providers/redhat/infra_manager/event_parsing/builder.rb

```diff
@@ -0,0 +1,15 @@
+module ManageIQ::Providers::Redhat::InfraManager::EventParsing
```

Comment:
This looks almost the same as module `ManageIQ::Providers::Redhat::InfraManager::Inventory::Builder`. The only difference is `strategy_model`. I wonder whether we could reuse it.